File: Treinamento/C/Estrutura_de_repetição/Questão06.c
#include <stdio.h>

int main() {
    int n;

    printf("Digite o número desejado: ");
    scanf("%i", &n);

    for (int c = 1; c <= 10; c++) {
        printf("%i X %i = %i\n", n, c, n * c);
    }
}

File: Treinamento/Java/Estrutura_de_repetição/Questão06.java
import java.util.Scanner;
public class Questão06 {
    public static void main(String[] args) {
        int soma = 0;
        Scanner s;
        s = new Scanner(System.in);

        for (int c = 0; c < 5; c++) {
            System.out.print("Digite o número desejado: ");
            int n = s.nextInt();
            soma += n;
        }
        
        double media = soma / 5.0;
        System.out.println("Soma: " + soma);
        System.out.printf("Média: %.2f\n", media);
    }
}

